### ISSUE TITLE
Burundi: Need map of PLHIV & ART Coverage by Monday for COP mtgs

### ISSUE THREAD

**Issue by gsarfaty** [USAID-OHA-SI/TeamTracking#106]
Request received to map plhiv & ART coverage by tomorrow for Burundi's Jo'burg Meetings. Noah was in CC and has their data. If he can send it to me I can do it, or if he wants to use his mapping skills, I can defer to him. I'm hand tied until I receive data.

**Comment by noahbartlett** [comment]
I can’t access Arc GiS unless I am on the network at CP3. Leaving this for Gina.

**Comment by jaliasd** [comment]
Feel free to keep me in the loop. What data do you need? I can ask the Burundi team here.

**Comment by gsarfaty** [comment]
Thanks! They've been messaging actively and Noah sent me their most recent DP to work from. Details with links will be updated here so you have all the info.

**Comment by gsarfaty** [comment]
Team wants plhiv & art coverage by peds, adult males, adult females.
Data pack and datasets extracted from it for the above mentioned populations are in this [folder](https://drive.google.com/drive/folders/1XyKxJYk2SMfViac0m4RKR72MZHvCOdGT).

Code for extracting from DP and creating these is [here](https://github.com/gsarfaty/Projects/blob/master/COP19/Burundi_DP_PLHIV_ArtCoverage.R).

@jaliasd or @noahbartlett if you can QC the tables before I create visuals with them, that would be useful. Let me know if you have time. I will break for an hour or so before moving forward to allow time for this if anyone is able.

**Comment by gsarfaty** [comment]
PEPFAR Burundi Provincial shapefile, BurundiProvinceLsib2016May, is missing Rumonge Province (known issue). I created an updated provincial file for Burundi by assigning the following communes to Romonge Province:
- Buyengero
- Bugarama
- Rumonge
- Burambi
- Muhuta

After assigning these to Romonge in the commune level shapefile, I dissolved on province name to create the updated provincial shp. I assigned DATIM uids to each province. This new shp, BurundiProvinceLsib2019March, has been added to the above linked folder and will be used for mapping.

Will also liaise w/Nate & S/GAC systems to get new boundary file loaded to DATIM, Pano, and AGOL.
